src/qt/sendcoinsdialog.cpp

```diff
@@ -277,8 +277,11 @@ void SendCoinsDialog::on_sendButton_clicked()
     QStringList formatted;
     for (const SendCoinsRecipient &rcp : currentTransaction.getRecipients())
     {
-        // generate bold amount string
+        // generate bold amount string with wallet name in case of multiwallet
         QString amount = "<b>" + BitcoinUnits::formatHtmlWithUnit(model->getOptionsModel()->getDisplayUnit(), rcp.amount);
+        if (model->isMultiwallet()) {
+            amount.append(" <u>"+tr("from wallet %1").arg(GUIUtil::HtmlEscape(model->getWalletName()))+"</u> ");
```

Comment:
Not sure about the underline, it's already bold.

Reply:
I think its okay since the used wallet is a pretty critical information. Overhauling the send-confirmation dialog is something that would be possible in the long run.